packages/language-server-tests-benchmarks/src/benchmarks/css/document_symbols.bench.ts

```diff
@@ -0,0 +1,32 @@
+import { afterAll, bench, describe } from "vitest";
+import { startLanguageServer } from "../../server";
+import { fileURLToPath } from "url";
+
+const filePath = fileURLToPath(
+	new URL("../../../fixture/document_symbols_benchmark.css", import.meta.url)
+);
+
+const weblsp = await startLanguageServer(undefined, "weblsp");
+const weblspUri = (await weblsp.openTextDocument(filePath, "css")).uri;
+
+const vscodeLsp = await startLanguageServer(undefined, "vscode-css");
+const vscodeLspUri = (await vscodeLsp.openTextDocument(filePath, "css")).uri;
+
+describe("DocumentSymbols", async () => {
+	bench("weblsp - DocumentSymbols", async () => {
+		await weblsp.sendDocumentSymbolRequest(weblspUri);
+	});
+
+	if (!process.env.CODSPEED) {
+		bench("vscode-css-languageserver - DocumentSymbols", async () => {
+			await vscodeLsp.sendDocumentSymbolRequest(vscodeLspUri);
+		});
+	}
```

Comment:
```suggestion
describe("Document Symbols", async () => {
	bench("weblsp - Document Symbols", async () => {
		await weblsp.sendDocumentSymbolRequest(weblspUri);
	});

	if (!process.env.CODSPEED) {
		bench("vscode-css-languageserver - Document Symbols", async () => {
			await vscodeLsp.sendDocumentSymbolRequest(vscodeLspUri);
		});
	}
```
names here are human readable